code/modules/mob/living/carbon/human/species/station/station.dm

```diff
@@ -91,6 +91,11 @@
 		"Your scales bristle against the cold."
 		)
 
+/datum/species/unathi/handle_post_spawn(var/mob/living/carbon/human/H)
+	..()
+	if(H.gender == "male")
+		breakcuffs = 1
```

Comment:
This changes the species' own breakcuffs variable. You probably meant `H.breakcuffs = 1`?